dotenv-linter/src/schema.rs

```diff
@@ -0,0 +1,384 @@
+use anyhow::Result;
```

Comment:
Looks like we don't need anyhow. That's the only usecase and we can keep it simpler for now. 

I see only two possible kind of errors for now:
- io::Error (schema file handling)
- serde

Since it's limited number of errors and they have different meaning, I think it would be better to avoid extra dependency here. 
I'd suggest to use either:
- [thiserror](https://crates.io/crates/thiserror)  - it's proc (derive) macro, so dependency used only for compilation
- manually define a enum with display

I'd go with thiserror, and would create an error like that:

```rs
enum SchemaReadError {
  #[error("failed to read file: {0}")]
  Read(#[from] std::io::Error),
  #[error("schema format is invalid: {0}")]
  Invalid(#[from] serde_json::Error)
}
```

Thus, error messages will be more controllable and we avoid runtime-dependency (moreover, anyhow uses boxing)

Reply:
i will take out anyhow. Its my automatic include in all projects (I think it should be in std quite honestly)